DuckDuckGoTests/MaliciousSiteProtection/MaliciousSiteProtectionFeatureFlagsTests.swift

```diff
@@ -0,0 +1,112 @@
+//
+//  MaliciousSiteProtectionFeatureFlagsTests.swift
+//  DuckDuckGo
+//
+//  Copyright © 2024 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Testing
+import BrowserServicesKit
+@testable import DuckDuckGo
+
+@Suite("Malicious Site Protection - Feature Flags", .serialized)
+final class MaliciousSiteProtectionFeatureFlagsTests {
+    private var sut: MaliciousSiteProtectionFeatureFlags!
+    private var featureFlaggerMock: MockFeatureFlagger!
+    private var configurationManagerMock: PrivacyConfigurationManagerMock!
+
+    init() async throws {
+        featureFlaggerMock = MockFeatureFlagger()
+        configurationManagerMock = PrivacyConfigurationManagerMock()
+        sut = MaliciousSiteProtectionFeatureFlags(featureFlagger: featureFlaggerMock, privacyConfigManager: configurationManagerMock)
+    }
+
+    deinit {
```

Comment:
now that I think of nilling properties in deinit is probably not needed as it will do it automatically. I will fix it.